k8s-readonly.tf

```diff
@@ -37,6 +37,44 @@ resource "kubernetes_deployment" "apiary_hms_readonly" {
       }
 
       spec {
+        init_container {
```

Comment:
can we make init container optional, to not run when using external_database_host